node_modules/oae-admin/tenants/tenants.html

```diff
@@ -121,8 +42,99 @@
             </div>
```

Comment:
It would be nice if we could make the `Create tenant` button an `oae-list-action` (see screenshot from Discussions) and turn the actual tenant creation form into a modal. That allows us to split out the logic of listing and creating, which will make it a bit more manageable.

<img width="1404" alt="screen shot 2015-09-10 at 09 28 35" src="https://cloud.githubusercontent.com/assets/109850/9794185/90314b02-579e-11e5-8caa-c28fca331f29.png">